pkg/controller/m3admin_client.go

```diff
@@ -76,19 +81,26 @@ func clusterURLProxy(cluster *myspec.M3DBCluster) string {
 	return url
 }
 
-func newMultiAdminClient(m3adminClient m3admin.Client, logger *zap.Logger) *multiAdminClient {
+func newMultiAdminClient(adminOpts []m3admin.Option, logger *zap.Logger) *multiAdminClient {
```

Comment:
should the logger just be another another `m3admin.Option`?

Reply:
Actually I just realized we need it because we pass the option to the namespace/placement clients which take a different `Option` type than `m3admin.Option`

Reply:
Well that's annoying